Packs/CommonScripts/Scripts/SearchIncidentsV2/SearchIncidentsV2.py

```diff
@@ -142,6 +145,18 @@ def search_incidents(args: Dict):   # pragma: no cover
         if args.get('trimevents') == '0':
             args.pop('trimevents')
 
+    if args.get('includeinformational'):
+        if not ENABLE_INCLUDEINFORMATIONAL_ARG:
+            raise ValueError('The includeinformational argument supported only in XSIAM from version x.x.x.')
+        if not (args.get('fromdate') and args.get('todate')):
+            raise ValueError('The includeinformational argument requires fromdate and todate arguments.')
+        if (datetime.utcnow() - fromdate).total_seconds() > 5 * 60 * 60:
+            args['fromdate'] = arg_to_datetime('5 hours ago').isoformat()
+            demisto.log(f'fromdate: {fromdate} is more than 5 hours from now: {datetime.utcnow()}. '
```

Comment:
We use demisto.info or demisto.debug (This will not be presented to the user unless he will look at the logs. Please change this to a readable output.

Reply:
when I use demisto.info I dont see the log in the war room UI, are you sure that this needs to be changed? 
demisto.log seems to produce the desired result